packages/zowe-explorer/src/trees/dataset/DatasetTree.ts

```diff
@@ -154,13 +163,20 @@ export class DatasetTree extends ZoweTreeProvider<IZoweDatasetTreeNode> implemen
                 if (entry == null) {
                     if (sourceNode.contextValue === Constants.DS_MEMBER_CONTEXT) {
                         dsname = destUri.path.match(/^\/[^/]+\/(.*?)\/[^/]+$/)[1] + "(" + (sourceNode.getLabel() as string) + ")";
-                        await ZoweExplorerApiRegister.getMvsApi(destinationInfo.profile).createDataSetMember(dsname, {});
+                        // Detect binary and set allocation attributes
+                        const allocAttrs = SharedContext.isBinary(sourceNode)
+                            ? { recordFormat: "U", lrecl: 0, blksize: 32760 }
+                            : {};
```

Comment:
I could be missing something here, but can we detect if the data set is binary by checking if `entry.encoding` is equal to `{ kind: "binary" }`?